client/blocks/product-purchase-features-list/test/video-audio-posts.jsx

```diff
@@ -45,28 +50,22 @@ describe( 'VideoAudioPosts should use proper description', () => {
 		},
 	};
 
-	[ PLAN_BUSINESS, PLAN_BUSINESS_2_YEARS ].forEach( ( plan ) => {
-		test( `for business plan ${ plan }`, () => {
-			const comp = shallow( <VideoAudioPosts { ...props } plan={ plan } /> );
-			expect( comp.find( 'PurchaseDetail' ).props().description ).toContain( 'Business Plan' );
-		} );
+	test.each( [ PLAN_BUSINESS, PLAN_BUSINESS_2_YEARS ] )( `for business plan %s`, ( plan ) => {
+		render( <VideoAudioPosts { ...props } plan={ plan } /> );
+		expect( screen.queryByTestId( 'purchase-detail' ) ).toHaveTextContent( /Business Plan/ );
 	} );
 
-	[ PLAN_ECOMMERCE, PLAN_ECOMMERCE_2_YEARS ].forEach( ( plan ) => {
-		test( `for ecommerce plan ${ plan }`, () => {
-			const comp = shallow( <VideoAudioPosts { ...props } plan={ plan } /> );
-			expect( comp.find( 'PurchaseDetail' ).props().description ).toContain( 'Ecommerce Plan' );
-		} );
+	test.each( [ PLAN_ECOMMERCE, PLAN_ECOMMERCE_2_YEARS ] )( `for ecommerce plan %s`, ( plan ) => {
+		render( <VideoAudioPosts { ...props } plan={ plan } /> );
+		expect( screen.queryByTestId( 'purchase-detail' ) ).toHaveTextContent( /Ecommerce Plan/ );
 	} );
 
-	[ PLAN_PREMIUM, PLAN_PREMIUM_2_YEARS ].forEach( ( plan ) => {
-		test( `for premium plan ${ plan }`, () => {
-			const comp = shallow( <VideoAudioPosts { ...props } plan={ plan } /> );
-			expect( comp.find( 'PurchaseDetail' ).props().description ).toContain( '13 GB of media' );
-		} );
+	test.each( [ PLAN_PREMIUM, PLAN_PREMIUM_2_YEARS ] )( `for premium plan %s`, ( plan ) => {
+		render( <VideoAudioPosts { ...props } plan={ plan } /> );
+		expect( screen.queryByTestId( 'purchase-detail' ) ).toHaveTextContent( /13 GB of media/ );
```

Comment:
Only thought here is that the "13 GB" portion could change over time, so I wonder if we should let it be more flexible

Reply:
I think that's fine, considering that it tests what the user sees. If we change it, the test will fail, and we'll need to update it to confirm that the change was intentional.